Form-Controls/index.html

```diff
@@ -1,28 +1,74 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="This is a form of ordering T-Shirt" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>T-Shirt Order Form</title>
+  </head>
+  <body>
+    <div class="form-container">
+      <h1 class="form-title">Order Your T-Shirt</h1>
+      <form class="order-form" id="orderForm">
+        <div class="form-group">
+          <label class="form-label" for="name">Name</label>
+          <input
+            type="text"
+            class="form-input"
+            id="name"
+            name="name"
+            required
+          />
```

Comment:
You forgot to validate `name` as a text string of two characters or more. Have a read about these attributes - [minlength](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/minlength) or [pattern](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern)